cve_bin_tool/cvedb.py

```diff
@@ -357,6 +398,24 @@ def init_database(self) -> None:
             cursor.execute("DROP TABLE cve_exploited")
             cursor.execute(exploit_table_create)
 
+        # Check schema on cve_metrics
+        if not self.latest_schema("cve_metrics", cve_metrics_schema, cursor):
+            self.LOGGER.info("Upgrading cve_exploited data. This may take some time.")
+            self.LOGGER.info(
+                "If this step hangs, try using `-u now` to get a fresh db."
+            )
+            cursor.execute("DROP TABLE cve_metrics")
+            cursor.execute(cve_metrics_table_create)
+
+        # Check schema on metrics
+        if not self.latest_schema("metrics", metrics_schema, cursor):
+            self.LOGGER.info("Upgrading cve_exploited data. This may take some time.")
```

Comment:
The comment in line 403/412 should be cve_metrics data, However, we aren't actually downloading the data at this stage, just creating the table

Reply:
That is why I haven't included the download function. I have recently updated the table schema and made updates wherever it is referenced or utilized.